tools/swift-format/Sources/Rules/VarDeclSyntax+Convenience.swift

```diff
@@ -13,21 +13,21 @@ extension VariableDeclSyntax {
   }
 
   /// Returns the first identifier.
-  var identifier: IdentifierPatternSyntax {
+  var firstIdentifier: IdentifierPatternSyntax {
     return identifiers[0]
   }
 
-  /// Returns the type of the variable, if explicitly stated in the declaration.
-  var type: TypeSyntax? {
+  /// Returns the first type explicitly stated in the declaration, if present.
+  var firstType: TypeSyntax? {
     for binding in bindings {
```

Comment:
You still need to update the implementations of these functions 🙂 

I think they can be expressed as simply as `return binding.first?.typeAnnotation?.type`, and `return binding.first?.initializer` to return the corresponding thing from the first binding if it's present or nil if it's not.